Documentation/building/linux-instructions.md

```diff
@@ -158,3 +158,63 @@ Finally, let's build coreclr with updated clang/llvm. If you meet a lldb related
 lgs@ubuntu time ROOTFS_DIR=/work/dotnet/rootfs-coreclr/arm ./build.sh arm release clean cross 
 ```
 
+Additional optimization levels for ARM/Linux: -Oz and -Ofast
+============================================================
+
+This instruction is to enable -Oz or -Ofast optimization level of clang on ARM/Linux. There are the optimization levels of clang/LLVM that have similar options with optimization flags of Microsoft Visual Studio 2015 compiler. The below table shows what we have to enable for the code optimization of the CoreCLR run-time either the size or speed on embedded devices. 
+
+| **Content** | **Visual Studio (Windows)** | **Clang/LLVM (Linux)** |
+| --- | --- | --- |
+|To Disable Optimization| /Od | -O0 (debug build)|
+|To Minimize Size| /O1 | -Oz |
+|To Maximize Speed| /O2  | -Ofast |
+|To Support Full Optimization| /Ox | -O3 (release build) |
```

Comment:
http://stackoverflow.com/questions/15548023/clang-optimization-levels has good summary.

The extra optimizations in Ofast vs. O3 are mostly around floating point math that is pretty much non-existent in CoreCLR.